NodeJS 18x support (#518)
heremaps/here-data-sdk-typescript: CHANGELOG.md

## v2.0.0 (13/05/2024)

> - Were stopped supporting nodejs versions less than 18.x.
> - Remove node_fetch from the dependencies.

Reviewer: ```suggestion
- Discontinued support of Node.js versions less than 18.x.
- Removed `node_fetch` from the dependencies.
```
